.github/workflows/e2e-tests.yml

```
@@ -3,9 +3,11 @@ on:
     inputs:
       goodmap-version:
         required: true
+        default: main
```

actionlint (1.7.10)

input "goodmap-version" of workflow_call event has the default value "main", but it is also required. if an input is marked as required, its default value will never be used

(events)